### PR TITLE
Branching based on Labreview branch

### PR DESCRIPTION
**Pull request by Singhal-Vaibhav**

### CODE REVIEW

File: python/foglamp/plugins/south/game/game.py [hunk 1]
```diff
@@ -259,4 +261,3 @@ def plugin_shutdown(handle):
     """
     _plugin_stop(handle)
```
Comment (amarendra-dianomic) [inline python/foglamp/plugins/south/game/game.py]: Not needed. Can be removed.

**Review by amarendra-dianomic — DISMISSED** [review]
1) remove pexpect exception from line#211
2) change "handle" to "new_config" at line#240
3) Remove _plugin_stop() and its call.
4) restart="yes" at line#238

File: python/foglamp/plugins/south/game/game.py [hunk 2]
```diff
@@ -233,30 +235,18 @@ def plugin_reconfigure(handle, new_config):
     # Plugin should re-initialize and restart if key configuration is changed
     if 'pollInterval' in diff:
         new_handle = copy.deepcopy(new_config)
```
Comment (ashish-jabble) [inline python/foglamp/plugins/south/game/game.py]: Below is the right order to re-configure
```
plugin_shutdown(handle) 
new_handle = plugin_init(new_config)
new_handle['restart'] = 'yes'
```

File: python/foglamp/plugins/south/game/game.py [hunk 3]
```diff
@@ -109,104 +111,104 @@ def plugin_poll(handle):
...
-                        "red": _RED_POINTS,
-                        "green": 0.0,
-                        "blue": 0.0,
-                        "accelerometer": 0.0,
-                        "lateral": 0.0,
-                        "flip": 0.0
-                    }
-                })
+                'asset': 'game/points',
+                'timestamp': time_stamp,
```
Comment (ashish-jabble) [inline python/foglamp/plugins/south/game/game.py:118]: To make consistency for plugins use `utils.local_timestamp()`

**Review by ashish-jabble — CHANGES_REQUESTED** [review]

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**